en/cookbook/refactoring-multilang.rst

```diff
@@ -0,0 +1,100 @@
+Refactoring Multilanguage Documents
+===================================
+
+Documents that are :doc:`multilanguage <../reference/multilang>` store a copy
+of their translated fields for each locale. When you make refactor documents
+and fields become translated or should no longer be translated, the data in the
+PHPCR needs to be migrated.
+
+A command line tool and a class are provided to help with this task.
+
+.. versionadded:: 1.3
+    The command and helper class where introduced in PHPCR-ODM 1.3.
+
+Procedure
+---------
+
+.. warning::
+
+    As always with data migrations, create a backup before you attempt to
+    migrate the live data. If there is a bug or you make a mistake, data
+    could be destroyed otherwise.
+
+The first step is to update the translation metadata (add or remove the
+translated attribute of a field mapping, or change the translation strategy
+declaration on the class mapping). The conversion tool converts to the current
+state. Sometimes it can guess the previous state, sometimes you will need to
+provide that information.
+
+Once this is done, use the command line
+tool ``doctrine:phpcr:document:convert-translation``. If you need additional
+logic, you can also write your own PHP code that instantiates
+``Doctrine\ODM\PHPCR\Tools\Helper\TranslationConverter`` and calls its
+``convert`` method. If you do the later, make sure to read the phpdoc on the
+``convert`` method.
+
+Converting Untranslated Fields to be Translated
+-----------------------------------------------
+
+First, add the ``translated`` mapping attribute for the fields that should
+become translated. If the document had no translated fields previously, you
+also need to define a translator strategy on the class mapping, as explained
+in :doc:`multilanguage support <../reference/multilang>`.
+
+Now you can run the command to make fields translated. It will copy the
+current untranslated value of the fields into all locales that are configured
+in the :ref:`LocaleChooser <multilang_chooser>`. From this, editors can adjust
+translations in specific languages.
```

Comment:
we have no way of knowing which fallbacks the user configured or not. i
think what we should do is require the user to specify this explicitly.